packages/cli/src/hooks/init/inithook.ts

```diff
@@ -48,6 +48,8 @@ const hook: Hook<'init'> = async function (opts) {
         this.log(chalk.blue('bf config:telemetry:enable'))
       }
 
+      fs.mkdirpSync(this.config.configDir)
+
       await fs.writeFileSync(path.join(this.config.configDir, 'config.json'), JSON.stringify(userConfig, null, 2))
```

Comment:
This line, although not made by you. Feel free to update it.

```suggestion
      await fs.writeFile(path.join(this.config.configDir, 'config.json'), JSON.stringify(userConfig, null, 2))
```